hail/python/hailtop/batch/batch.py

```diff
@@ -242,7 +248,6 @@ def read_input_group(self, **kwargs: str) -> ResourceGroup:
         ...                            bim="data/example.bim",
         ...                            fam="data/example.fam")
         >>> j = b.new_job()
-        >>> j.command(f"plink --bfile {bfile} --geno --make-bed --out {j.geno}")
```

Comment:
This removes a critical example of using `bfile` without any extension, just add `  # doctest: +SKIP` to the end of this line.